lib/autobot.ts

```diff
@@ -164,8 +157,13 @@ export class Autobot {
         logger.debug(`${prPlugins.length} ${scope} plugins initialized`);
 
         return prPlugins.forEach(meta => {
-          meta.plugin.apply(this.hooks.pr, context);
-          this.plugins[meta.name] = meta;
+          if (
+            meta.plugin.actions === "*" ||
+            meta.plugin.actions.includes(context.payload.action as PullRequestAction)
```

Comment:
A PR plugin now has the concept of an action which aligns the the webhook [actions](https://developer.github.com/v3/activity/events/types/#pullrequestevent) field. By default it accepts all, but the implementing class can override it to only run on a subset of actions.